src/components/Socials/styles.scss

```
@@ -1,3 +1,64 @@
+// Mobile portrait
 .wrapper {
+	display: flex;
+	justify-content: center;
+	width: 100%;
 	font-size: 1em; //nothing comment to make sure class appears
+	
+		li {
```

```suggestion
		.item {
```
we can give it this in the `renderUtils` - in `renderItem` I think.